src/lib/components/Image/Image.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactTestRenderer from 'react-test-renderer';
+import Image from './Image';
+
+describe('Image component', () => {
+  it('should render a default Image correctly', () => {
+    const image = ReactTestRenderer.create(
+      <Image src="http://placekitten.com/g/300/300" alt="" />);
+    const json = image.toJSON();
+    expect(json).toMatchSnapshot();
+  });
+
+  it('should render Image with props correctly', () => {
+    const image = ReactTestRenderer.create(
+      <div>
+        <Image src="http://placekitten.com/g/300/300" alt="" />
+        <Image bordered src="http://placekitten.com/g/300/300" alt="" />
+        <Image shadowed src="http://placekitten.com/g/300/300" alt="" />
+        <Image bordered shadowed src="http://placekitten.com/g/300/300" alt="" />
```

Comment:
It would be better to split these into separate tests so its clear what is broken when tests fail.